app/views/users/view.scala.html

```diff
@@ -110,6 +110,13 @@ <h1 class="username">
                                         data-placement="top" title="@messages("user.pgp.pubKey")"></i>
                                 </span>
                             }
+
+                            @if(users.current.isDefined && users.current.get.can(ViewAdmin).in(user)) {
```

Comment:
`users.current.get can ViewAdmin in user` would be more consistent with the rest of the code